RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -0,0 +1,182 @@
+                {
+                    bool isGoverningToken = output.AssetId.Equals(Blockchain.GoverningToken.Hash);
+                    if (isGoverningToken || output.AssetId.Equals(Blockchain.UtilityToken.Hash))
+                    {
+                        // Add new unspent UTXOs by account script hash.
+                        UserUnspentCoinOutputs outputs = _userUnspentCoins.GetAndChange(
+                            new UserUnspentCoinOutputsKey(isGoverningToken, output.ScriptHash, tx.Hash),
+                            () => new UserUnspentCoinOutputs());
+                        outputs.AddTxIndex(outputIndex, output.Value);
+                    }
+                    outputIndex++;
+                }
+
+                // Iterate all input Transactions by grouping by common input hashes.
+                foreach (var group in tx.Inputs.GroupBy(p => p.PrevHash))
+                {
+                    TransactionState txPrev = snapshot.Transactions[group.Key];
+                    // For each input being spent by this transaction.
+                    foreach (CoinReference input in group)
+                    {
+                        // Get the output from the the previous transaction that is now being spent.
+                        var outPrev = txPrev.Transaction.Outputs[input.PrevIndex];
+
+                        bool isGoverningToken = outPrev.AssetId.Equals(Blockchain.GoverningToken.Hash);
+                        if (isGoverningToken || outPrev.AssetId.Equals(Blockchain.UtilityToken.Hash))
+                        {
+                            // Remove spent UTXOs for unspent outputs by account script hash.
+                            var userUnspentCoinOutputsKey =
+                                new UserUnspentCoinOutputsKey(isGoverningToken, outPrev.ScriptHash, input.PrevHash);
+                            UserUnspentCoinOutputs outputs = _userUnspentCoins.GetAndChange(
+                                userUnspentCoinOutputsKey, () => new UserUnspentCoinOutputs());
+                            outputs.RemoveTxIndex(input.PrevIndex);
+                            if (outputs.AmountByTxIndex.Count == 0)
+                                _userUnspentCoins.Delete(userUnspentCoinOutputsKey);
+                        }
+                    }
+                }
+            }
+        }
+
+        public void OnCommit(Snapshot snapshot)
+        {
+            _userUnspentCoins.Commit();
+            _db.Write(WriteOptions.Default, _writeBatch);
+        }
+
+        public bool ShouldThrowExceptionFromCommit(Exception ex)
+        {
+            return true;
+        }
+
+        public void PreProcess(HttpContext context, string method, JArray _params)
+        {
+        }
+
+        public JObject OnProcess(HttpContext context, string method, JArray _params)
+        {
+            if (method != "getunspents") return null;
+
+            UInt160 scriptHash = UInt160.Parse(_params[0].AsString());
```

Comment:
Can we add support for address here, such as https://github.com/neo-project/neo-plugins/pull/43/commits/782d77726ee5b60bd894937e18b399dc0c8a157f?

Reply:
done.